mpas_analysis/obs/observational_datasets.xml

```diff
@@ -495,7 +524,7 @@
       - indexNino34
     </tasks>
     <subdirectory>
-      Ocean/Nino/HadISST
+      Ocean/Nino
```

Comment:
These subdirectories were necessary to extract a separate `README.md` and `obs.bib` for the two sets of Nino34 observations.  Without the subdirectory, the files for one get overwritten by the files for the other.  Please change it back or suggest an alternative approach.